stabilizer/src/main/java/com/hazelcast/stabilizer/agent/AgentCli.java

```diff
@@ -13,14 +13,13 @@
 
     private final OptionSpec<String> cloudIdentitySpec = parser.accepts("cloudIdentity",
             "Cloud identity")
-            .withRequiredArg().ofType(String.class);
-
+            .withOptionalArg().ofType(String.class);
```

Comment:
Why is this field optional? If you provide --cloudIdentity, you need to pass in a file. Now it is possible to call --cloudIdendity without a file.. which doesn't make much sense to me. Perhaps I'm missing something.